tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -82,36 +82,51 @@ steps:
       echo Tag: $tag
       cp .npmrc ~/.npmrc
       t1=0
-      sortedPackages=$(cat $(Pipeline.Workspace)/pack/packagePublishOrder.txt)
-      for packageName in $sortedPackages
+      if [[ -f $(Pipeline.Workspace)/pack/packagePublishOrder.txt ]]; then
+        packages=$(cat $(Pipeline.Workspace)/pack/packagePublishOrder.txt)
+        sorted=true
+        echo "sorted"
+      else
+        packages=*.tgz
+        sorted=false
+        echo "not sorted"
+      fi
+      echo $sorted
```

Comment:
Nit: consolidate the `echo`s, I think the last one is enough if it also says what the true/false means.

Reply:
I'm actually using these echos for testing because I saw it didn't publish anything 😄. I'll get rid of them once I figure it out.